src/Control/Monad/Aff.purs

```diff
@@ -75,17 +76,22 @@ cancelWith aff c = runFn3 _cancelWith nonCanceler aff c
 -- | If you do need to handle exceptions, you can use `runAff` instead, or
 -- | you can handle the exception within the Aff computation, using
 -- | `catchError` (or any of the other mechanisms).
-launchAff :: forall e a. Aff e a -> Eff (err :: EXCEPTION | e) Unit
-launchAff = runAff throwException (const (pure unit)) <<< liftEx
-  where
-  liftEx :: Aff e a -> Aff (err :: EXCEPTION | e) a
-  liftEx = _unsafeInterleaveAff
+launchAff :: forall e a. Aff (err :: EXCEPTION | e) a -> Eff (err :: EXCEPTION | e) Unit
```

Comment:
Since this will be included in a breaking release anyway (due to 0.9 updates), it's a good chance to clean up things like this. I'd rather a user `void` a return value they don't want, then not have the option (or multiply the surface area of the API).